From 46584788f5ef58a53c29bf3fb4de790c2cb383bb Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Thu, 1 Mar 2018 09:48:05 +0800
Subject: [PATCH 3471/5242] MLK-17603 arm: dts: imx7d-sdb-epdc: enable enet1
 for epdc extra dts file

commit  46e6a4fa72011c901a077ac8b810d0f510e531fb from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable enet1 for epdc extra dts file since only enet2 has pin conflict
with epdc.

Reviewed-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb-epdc.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb-epdc.dtsi b/arch/arm/boot/dts/imx7d-sdb-epdc.dtsi
index cbc7ca1..604bbbc 100644
--- a/arch/arm/boot/dts/imx7d-sdb-epdc.dtsi
+++ b/arch/arm/boot/dts/imx7d-sdb-epdc.dtsi
@@ -11,7 +11,7 @@
 };
 
 &fec1 {
-	status = "disabled";
+	status = "okay";
 };
 
 &fec2 {
-- 
1.7.9.5

